test/UID2.Client.Test/EncryptionTestsV3.cs

```diff
@@ -32,6 +32,154 @@ public void SmokeTest()
             Assert.Equal(EXAMPLE_UID, res.Uid);
         }
 
+        //////////////////////  Sharing tests
+        [Fact]
+        public void CanEncryptAndDecryptForSharing()
```

Comment:
this test verifies if the same client instance can encrypt and decrypt. shall we have other tests with 2 clients and 1 client encryption's can be decrypted by another client instance's decryption call? (or that part of sharing is out of the scope of this MR)?

Reply:
done, thanks